ruff.toml

```diff
@@ -0,0 +1,5 @@
+line-length=88
+exclude = [".git","__pycache__"]
+
+[lint]
+ignore= ["E741"]
```

Comment:
You'll also want to have a `select` array here of rules to include, otherwise I don't think `ruff` will have any rules to check against.

From the docs, a reasonable starting place that isn't too picky:
```
select = [
  # pycodestyle
  "E",
  # Pyflakes
  "F",
  # pyupgrade
  "UP",
  # flake8-bugbear
  "B",
  # flake8-simplify
  "SIM",
  # isort
  "I",
]
```

telemetry-airflow uses 
```
select = [
    "E", # pycodestyle
    "W", # pycodestyle
    "F", # Pyflakes
    "B", # flake8-bugbear
    "C4", # flake8-comprehensions
    "D", # flake8-docstrings
    "I", # isort
    "SIM", # flake8-simplify
    "TCH", # flake8-type-checking
    "TID", # flake8-tidy-imports
    "Q", # flake8-quotes
    "UP", # pyupgrade
    "PT", # flake8-pytest-style
    "RUF", # Ruff-specific rules
]
```

Experimenter uses
```
select = [
    "A",
    "B",
    "C4",
    "DJ",
    "E",
    "F",
    "I",
    "ISC",
    "N",
    "PIE",
    "Q",
    "RET",
    "RSE",
    "RUF",
    "SIM",
    "TID",
    "UP",
    "W",
    "YTT",
    "PTH",
]
```

Reply:
Ah whoops.  Ended up going with the telemetry airflow setting, minus the docstring check because almost none of the test functions have them.  Created #213 to address this because I think it'd be good to enforce docstrings going forward.

Reply:
The only changes that are worth a closer look are where it replaced `.format` with an f-string.  A lot of them are used in queries and I don't know how well-tested the query statements are.  I looked through it and they all look fine but it could use a second pair of eyes.  Might also make sense to just make it into another PR at this point 🤷 

Also it left ~15 `.format` statements and I'm not sure why, don't know if there is any point to trying to get rid of them here or just let it go

Reply:
Ended up just doing E and F